02-main/ch2_analysis.tex

```diff
@@ -34,14 +34,14 @@ \section{Figures}
 
 This section contains examples of figures: \autoref{fig:galleria}, \autoref{fig:lorem}, \autoref{fig:ipsum}, \autoref{fig:dolor}, \autoref{fig:sit}
 
-\begin{figure}[H] 
+\begin{figure}[!ht] 
 	\centering 
 	\includegraphics[width=0.5\columnwidth]{galleria_stampe} 
-	\captionsource{A floating figure}{A floating figure: the lithograph \emph{Galleria di stampe}, of M.~Escher}{\url{http://www.mcescher.com/}}
+	\caption[A floating figure]{A floating figure: the lithograph \emph{Galleria di stampe}, of M.~Escher}
```

Comment:
I defined this command in the settings.tex. What was the issue with the display of the custom command in Overleaf?

Reply:
I honestly don't really remember. I switched to that since this is the default way for overleaf to handle captions. I now realize that it does not have the source, which does change the base idea of your command.

Reply:
Then we just have to revert this or fix issues with the default command if something is wrong with it. Then I can merge the PR.

Reply:
Hi, sorry for the delay, i forgot to fix this last issue. I tried reverting and it worked as intended, so i'm reverting this change